app/helpers/application_helper.rb

```diff
@@ -385,6 +385,12 @@ def show_parent?(obj)
     (obj.new_record? && obj.class.count > 0) || (!obj.new_record? && obj.class.count > 1)
   end
 
+  def documentation_button(section)
+    link_to(icon_text('question-sign', _('Documentation'), :class => 'icon-white'),
+            "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html##{section}",
```

Comment:
I suppose symlink would be problematic, obviously our 1.9 manual would report itself as 1.8 and all links would redirect to 1.8. So maybe adding some dummy page that would redirect to 1.8 would help?